### ISSUE TITLE
Extreme lagging drag performance of task window in all examples 

### ISSUE THREAD

**Issue by lubber-de** [bryntum/support#7096]
## Description

It seems some event handler is contantly consuming CPU when dragging any task detail window in any of the examples (i tried scheduler pro)
For example https://bryntum.com/products/schedulerpro/examples/effort/

Dragging task bars itself or anything else is super smooth as expected, but dragging any detail window is lagging performance making the drag feature almost impossible to use.

## Screenshot
![windowperformance](https://github.com/bryntum/support/assets/18379884/1ce827fe-27b7-4cce-974e-592360bf673e)

**Comment by lubber-de** [comment]
Yes, I depacked the schedulerpro 5.3.8 zip from the customerpanel on our local servers as is, and open the same examples from there (so still having a real webserver serving  the files. It also happens there and regardless of the example. Each time a task window is shown, the dragging is slow as seen in the screen capture

**Comment by ExtAnimal** [comment]
What happens when you use the Performance tab to see what's consuming the CPU?

The entire processing of a drag event is this. Just setting `translateXY`

![Screenshot 2023-06-30 at 16 51 24](https://github.com/bryntum/support/assets/200516/59d51e4f-163c-4f2f-9461-5d7f34cfe36a)

And looking at the Performance tab, each event takes 0.29 ms, and triggers no layout operations and no dropped frames.

**Comment by lubber-de** [comment]
Here is something from a capture while dragging the window
![image](https://github.com/bryntum/support/assets/18379884/749f3e58-c302-45b4-a713-c897d43a5f68)

If you need morespecific infos, please guide me, the devtools are huge :D

Out of curiosity, i tried the same on browserstack (because we dont have macs) to run it on Chrome114 on MacOS Big Sur.
From what i can tell it seems, on Mac it does _not_ happen.
I now also tried the examples on my private Win11 machine, where dragging also lags. (Our Business Notebooks run Win10). Tried chrome / edge and firefox: All react the same.

What i saw via devtools: You are using `transform: matrix()` while dragging, rather than `translateXY` (or even `translateXYZ` to force GPU acceleration)
Perhaps a reason?

**Comment by lubber-de** [comment]
Another idea: Perhaps it lags because the transform transition needs to end before the next "move" can be applied.
So we should try to add `transition: 0s` 

https://stackoverflow.com/questions/24209093/slow-rendering-while-dragging-a-div/24217076#24217076

**Comment by matsbryntse** [comment]
> Another idea: Perhaps it lags because the transform transition needs to end before the next "move" can be applied.

There's no transition of the transform, only opacity that doesn't change while dragging. We'll have to dig around a bit.

**Comment by lubber-de** [comment]
Seems to be finally fixed in 6.1.7

Duplicate of #4428
